miio/gateway.py

```diff
@@ -201,6 +218,38 @@ def discover_devices(self):
 
         return self._devices
 
+    def x_del(self, script_id):
+        """Delete script by id."""
+        return self.send("miIO.xdel", [script_id])
+
+    @command(click.argument("sid"), click.argument("command"))
+    def zigbee_command(self, sid, command):
+        self.discover_devices()
+        target = list(filter(lambda subdevice: subdevice.sid == sid, self.devices))
+        if len(target) < 1:
+            return f"Device with sid={sid} not found"
+        elif not hasattr(target[0], command):
+            return f"Device with sid={sid} has no method {command}"
+        else:
+            return getattr(target[0], command)()
+
+    def install_script(self, sid, action, builder):
```

Comment:
This function is actually install script to gateway. It doesn't subscribe to events... It can even install script which will callback to other device...